bciers/apps/registration1/tests/utils/customTransformErrors.test.ts

```diff
@@ -108,6 +108,19 @@ const phoneFormatError = [
   },
 ];
 
+const arrayMinItemsError = [
+  {
+    name: "minItems",
+    property: ".array_field",
+    message: "must NOT have fewer than 1 items",
```

Comment:
I changed this default error because all of our messages start with a capital letter and I felt like the NOT was yelling at us.